ci/test.py

```diff
@@ -41,6 +41,32 @@ def wrapper(self, *args, **kwargs):
         return func(self, *args, **kwargs)
     return wrapper
 
+def skipIfBaseImage(reason='functionality not supported on base images'):
```

Comment:
Perhaps this would be better as:
```python
 def skipIfNotImageType(image_type, reason='....')
 ...
    if self.test_image.rsplit('-')[-1] != image_type:
        self.skipTest(reason)
```

Reply:
fixed.